lua/img-clip/config.lua

```diff
@@ -136,6 +136,18 @@ M.get_option = function(key, opts)
     return nil
   end
 
+  if key == "template" and type(val) == "function" then
+    local placeholders = {
+      file_path = "$FILE_PATH",
+      file_name = "$FILE_NAME",
+      file_name_no_ext = "$FILE_NAME_NO_EXT",
+      cursor = "$CURSOR",
+      label = "$LABEL",
+    }
```

Comment:
So, what I wanted was to have the values in `placeholders` interpolated already at this point, before you call `val(placeholders)`. And in this case, I believe it is better to call the object "context" or similar instead of placeholders, to avoid confusing the placeholder names with the interpolated values.

Reply:
Does the latest commit work as intended?